portability-transfer/src/test/java/org/datatransferproject/transfer/copier/PortabilityInMemoryDataCopierTest.java

```diff
@@ -103,20 +102,15 @@ protected ExportResult<?> copyIteration(
     }
   }
 
-  @Parameterized.Parameters
   public static Iterable<PortabilityAbstractInMemoryDataCopier> data() {
     return Arrays.asList(
-        (new PortabilityAbstractInMemoryDataCopier[] {
-          Mockito.spy(new TestPortabilityInMemoryDataCopier()),
-          Mockito.spy(new TestPortabilityStackInMemoryDataCopier())
+        (new PortabilityAbstractInMemoryDataCopier[]{
+            Mockito.spy(new TestPortabilityInMemoryDataCopier()),
+            Mockito.spy(new TestPortabilityStackInMemoryDataCopier())
```

Comment:
optional nit: this isn't due to your PR, but since you're improving this test anyway: could you clarify the names of these two classes? eg: I don't know much about what's going on here, but it's not obvious to me why the additional parameter of a `JobStore` makes this spy a "stack" in-memory copier.

Reply:
I could see its not only the additional parameter of `JobStore` but the super classes of these test copiers add to the difference that make one of them the stack copier. `Portability**Stack**InMemoryDataCopier` vs `PortabilityInMemoryDataCopier`

`JobStore` is storing the `Stack<ExportInformation>` which is used in the `copy` method of `Portability**Stack**InMemoryDataCopier` to retrieve the stack for processing, while the "non-stack copier" doesn't require this field.